exist-versioning-release.md

```diff
@@ -119,11 +119,11 @@ You will require a system with:
 * A GPG key (for signing release artifacts)
 * A Java KeyStore with key (for signing IzPack Installer)
 * A valid Apple Developer Certificate (for signing Mac DMG)
+* A Github account and username / password or Github Personal access tokens (https://github.com/settings/tokens) with permission to publish Github releases to the eXist-db .org
 
 1. You will need login credentials for the eXist-db organisation on:
     1. Sonatype OSS staging for Maven Central - https://oss.sonatype.org/
-    2. Bintray - https://bintray.com/existdb
-    3. DockerHub - https://cloud.docker.com/orgs/existdb/
+    2. DockerHub - https://cloud.docker.com/orgs/existdb/
```

Comment:
Is it not worth mentioning that you need repo maintainer status on the eXist-db GitHub org (to replace the line about BinTray). I don't think just anyone can upload a release to GitHub, you need certain org/repo privileges I suspect.